### PR TITLE
Basic tests and CI

### LINKED ISSUE #6: Enable basic CI

### PR DESCRIPTION
**Pull request by przenz**
Resolves #6

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -36,6 +30,4 @@ app.get('/', (req, res) => {
      res.sendFile('index.html', {root: __dirname + '/public/'});
 });
 
-server.listen(process.env.PORT || 5000, () => {
-     console.log(`Server started on port ${process.env.PORT || 5000}`);
-});
+module.exports = server.listen(process.env.PORT || 5000);
```
Comment (przenz) [inline index.js:33]: This change is required for chai-http tests to work properly.

File: test/test.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+      newPlayer.disconnect();
+      socket.on('playerDisconnected', function(msg){
+        msg.should.have.lengthOf(20);
+        done();
+      })
+    })
+  })
+})
+
+describe('Server HTTP tests', function() {
```
Comment (przenz) [inline test/test.js:46]: If HTTP tests are running first, then socket tests does not pass, I'm unable to fix this and there is no useful information on the internet.

File: test/test.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+  , server = require('../index')
+  , io = require('socket.io-client')
+  , ioOptions = { 
+      transports: ['websocket']
+    , forceNew: true
+    , reconnection: false
+  }
+  , server
+  , socket
+  , testHost = 'http://localhost:5000/'
```
Comment (neu5) [inline test/test.js:14]: I'm not fun of this pattern but it's ok for now

File: test/test.js (hunk 2 above)
Comment (neu5) [reply test/test.js:46]: ok, ok

**Review by neu5 — APPROVED** [review]

**Pull request merged**